spec/rubocop/cop/style/if_unless_modifier_spec.rb

```diff
@@ -583,7 +584,7 @@ def f
   end
 
   context 'when Layout/LineLength is disabled' do
-    let(:line_length_config) { { 'Enabled' => false } }
+    let(:other_cops) { {} }
```

Comment:
How does this work? Isn't `LineLength` enabled by default in `config/default.yml`? If we override the `other_cops` definition in the beginning of this spec, I would expect RuboCop to use that default, and that contradicts to context title.

Reply:
Oh, interesting. I was under a different understanding, but you made me check my assumptions. Turns I was (mostly) correct! All cops are `'Enabled' => true` by default in the tests, including the disabled ones in `config/default.yml`. But they have **no** configurations:

```irb
> config.for_cop('Layout/LineLength')
 => {"Enabled"=>true}
> config.for_cop('Bundler/GemComment')
 => {"Enabled"=>true}
```

So without a `'Max'` setting, `Layout/LineLength` is effectively disabled.